euth/dashboard/templates/euth_dashboard/includes/project_form_information_tab.html

```diff
@@ -20,14 +20,10 @@ <h3 class="form-section-heading">{{ name }}</h3>
                 {{ field.label }} {% if field.field.required %}*{% endif %}
             </label>
             <p class='form-help-text'>{{ field.help_text }}</p>
-            {% if field|widget_type == 'checkboxinput' %}
-            <label>
-                {% get_checkbox_label form.project field.name as checkboxlabel %}
```

Comment:
I think this if in the tender logic (that checks the widget_type) could also be removed then.